db/models.py

```diff
@@ -46,3 +51,67 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-user"]
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/225978062-862d150c-47cc-45dd-a7e5-e61d7b819992.png)
are u sure that u should order by user?